### ISSUE TITLE
Hide the "Claim this group" button for the organizers of a given group

### ISSUE THREAD

**Issue by michaltrzesimiech** [rtcharity/eahub.org#617]

**Comment by Manoj-nathwani** [comment]
@michaltrzesimiech why is this needed? There's just some orgs we won't want anyone to ever own?

**Comment by michaltrzesimiech** [comment]
It's been raised as a request by a person that has just claimed a group and is now an organiser of it. She still sees the button and would prefer not to.

**Comment by sebgrebe** [comment]
Not sure that's a good idea. What if a group wants to add a second organiser?
Or do we want to add a "disable claim group btn" on the group edit panel?

**Comment by taymonbeal** [comment]
Any organizer of a group can add additional organizers as long as their profiles are public. It is indeed silly to show the claim button in this case, as it serves no useful function. I think at one point we talked about only showing the button for groups with no organizers, but it was argued that it's still useful for groups that have been abandoned. Ideally groups with active organizers wouldn't have it though.

**Comment by michaltrzesimiech** [comment]
The way I understand it is that we only want to hide it from the person that is the current Organiser of that group.

**Comment by taymonbeal** [comment]
Yeah, there are two separate issues. Hiding it from current organisers is straightforward. Separately, I think it would _also_ be good to hide it from _everyone_ if the group already has active organisers, but I'm not sure how we'd detect that.

**Comment by michaltrzesimiech** [comment]
I'd say we should keep the button visible to everyone but the current organisers. This will make it easier to claim a group once organisers leave without handing it over. Once organisers are there, we could make claim requests be sent to them as well.
